scripts/generator/templates/template-vaadin-bom.xml

```diff
@@ -15,7 +15,7 @@
     <properties>
 {{javadeps}}
         <slf4j.version>1.7.32</slf4j.version>
-        <jna.version>5.10.0</jna.version>
+        <jna.version>5.11.0</jna.version>
     </properties>
```

Comment:
Should slf4j be aligned as well? Other Platform releases like flow are on 1.7.36 (https://github.com/vaadin/flow/blob/5dc5f714bd4dc81083a3d382e0ea8fa0edde633b/pom.xml#L92)

Reply:
good catch, updated the slf4j.version